addon/components/value-viewer.js

```diff
@@ -46,4 +46,20 @@ export default Component.extend({
 
     return count;
   }),
+
+  indentation: computed("depth", function() {
+    let ret = "";
+    for (let i = 0; i <= this.get("depth"); ++i) {
+      ret += "  ";
+    }
+    return ret;
```

Comment:
Could use this instead:
```suggestion
    return "".padStart(this.get('depth'), " ");
```